include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -6065,6 +6066,20 @@ struct OrtApi {
   ORT_API2_STATUS(Node_GetAttributeByName, _In_ const OrtNode* node, _In_ const char* attribute_name,
                   _Outptr_result_maybenull_ const OrtOpAttr** attribute);
 
+  /** \brief Get the OrtNode's 'TENSOR' attribute as an OrtValue.
+   *
+   * \param[in] node The OrtNode instance.
+   * \param[in] attribute The OrtOpAttr instance.
+   * \param[out] attr_tensor Returns the newly created OrtValue if it's a 'TENSOR' attribute.
```

Comment:
maybe indicate that the output OrtValue is only created if the function is successful. e.g.,
```suggestion
   * \param[out] attr_tensor If successful, contains the 'TENSOR' attribute as a newly created OrtValue.
```